deployer/commands/transform/cost_table.py

```diff
@@ -86,12 +89,20 @@ def aws(
     # Sort the account names in alphabetical order
     df.sort_index(inplace=True)
 
+    # Transform months from 2024-01-01 to 2024-01
+    df.rename(
+        columns=lambda col: (
+            re.match("([0-9]*-[0-9]*)-[0-9]*", col).groups()[0]
+            if re.match("[0-9]*-[0-9]*-[0-9]*", col)
+            else col
+        ),
+        inplace=True,
+    )
+
```

Comment:
This was the key change I looked to make, so that the month columns look the same for AWS and GCP generated .csv files.